models.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose'
+
+
+export const ArtistDetail = mongoose.model('ArtisDetail', {
+ id: Number,
+ name: String,
+ years: String,
+ genre: String,
+ nationality: String,
+ bio: String,
+ wikipedia: String,
+ paintings: Number
```

Comment:
It will be safer for you if you add required: true to some fields. this will prevent from creating empty entries in the database ;)